hw/ip/csrng/data/csrng.hjson

```diff
@@ -268,6 +283,38 @@
         }
       ]
     },
+    {
+      name: "RECOV_ALERT_STS",
+      desc: "Recoverable alert status register",
+      swaccess: "rw0c",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "0",
+          name: "ENABLE_FIELD_ALERT",
+          desc: '''
+                This bit is set when the ENABLE field in the !!CTRL register is set to
+                a value other than 0x5 or 0xA.
+                Writing a zero to this register resets the status bits.
```

Comment:
Similar nitty question as I had on #8101 - I think "register" here should be "bit" and "status bits" should be "status bit" (repeated two more times below)

Reply:
Changing to be consistent, and on a bit basis.